cumulus_library/actions/builder.py

```diff
@@ -256,34 +211,52 @@ def run_matching_table_builder(
     :keyword builder: filename of a module implementing a TableBuilder
     :keyword db_parser: an object implementing DatabaseParser for the target database"""
     all_generators = manifest.get_all_generators()
+    matches = []
     for file in all_generators:
-        if builder and file.find(builder) == -1:
-            continue
-        _load_and_execute_builder(
-            config=config,
-            manifest=manifest,
-            filename=file,
-            db_parser=db_parser,
-        )
+        if builder and file.find(builder) != -1:
```

Comment:
should this be `if not builder or file.find():`? -- if the builder option is falsy, nothing gets included - is that what we want?

Reply:
yeah that might read better